### PR TITLE
Added target attribute inside anchor tags

### LINKED ISSUE #3293: Enhancing (UX) by directing external sites on new tab.
#### Description
Enhancing user experience (UX) by directing external sites on new tab. 

#### Current Behavior
On clicking the link to vote for the best Meshmap logo in the 'Goodbye, YAML' section, the layer 5 website reloads and opens twitter on the same tab, same behaviour can be observed on clicking the card 'JOIN THE CONVERSATION' in the community section, which opens the discuss.layer5.io site on the same tab. 

#### Desired Behavior
The desired behaviour is to direct the user to these websites on a new tab, when the respective links/cards are clicked. This creates a better user experience. 

#### Screenshots / Mockups
https://user-images.githubusercontent.com/107111616/194754758-a933a5b2-2559-42bc-99d9-6201d4d08316.mov

#### Environment:

- Host OS: Mac OS
- Browser: Chrome

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by sandeepB3**
Signed-off-by: Sandeep Pillai [sandeepxpillai@gmail.com](mailto:sandeepxpillai@gmail.com)

**Description**
Solved issue #3293 by adding target="_blank" attribute inside the respective anchor tags.

This PR fixes #3293 

**Notes for Reviewers**

Before:
**The video in issue #3293** 

After:

https://user-images.githubusercontent.com/107111616/194871321-31ef4a77-3a4d-4a49-b1b9-6520b73bd535.mov

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Meshmap/index.js [hunk 1]
```diff
@@ -74,9 +74,9 @@ const Meshmap = () => {
                   <img src={mesheryCloud} className="meshery-hero-img" alt="Meshery Cloud" />
                   <div className="draft-logo">
                     <p>Help choose the best logo for MeshMap:</p>
-                    <a href="https://twitter.com/layer5/status/1536832439833743363?s=20&t=oEDasGEBl8cvsA64P7NNTA">
+                    <a target="_blank" href="https://twitter.com/layer5/status/1536832439833743363?s=20&t=oEDasGEBl8cvsA64P7NNTA">
```
Comment (Nikhil-Ladha) [inline src/sections/Meshmap/index.js:77]: Add `rel="noreferrer"` attribute

File: src/sections/Meshmap/index.js [hunk 2]
```diff
@@ -74,9 +74,9 @@ const Meshmap = () => {
...
                       <img src={MeshMapDrafts} alt="MeshMap Draft" /></a><br />
-                    <p><a href="https://twitter.com/layer5/status/1536832439833743363?s=20&t=oEDasGEBl8cvsA64P7NNTA">Vote for your favorite design</a></p>
+                    <p><a target="_blank" href="https://twitter.com/layer5/status/1536832439833743363?s=20&t=oEDasGEBl8cvsA64P7NNTA">Vote for your favorite design</a></p>
```
Comment (Nikhil-Ladha) [inline src/sections/Meshmap/index.js:79]: Add `rel="noreferrer"` attribute

File: src/sections/Meshmap/index.js (hunk 1 above)
Comment (sandeepB3) [reply src/sections/Meshmap/index.js:77]: Made the required changes in a new PR: #3308

File: src/sections/Meshmap/index.js (hunk 2 above)
Comment (sandeepB3) [reply src/sections/Meshmap/index.js:79]: Made the required changes in a new PR: #3308

**Comment by sandeepB3** [comment]
Facing an issue in this branch, hence made the required changes in a new and updated branch. Here is the PR: #3308

**Pull request closed without merging**